src/router.js

```diff
@@ -64,287 +64,317 @@ import UserActivity from "./pages/UserActivity";
+          component={ClusterResourcesPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/services"
+          component={ServicesListPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/volumes"
+          component={PvsListPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/nodes"
+          component={ClusterNodes}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/pvcs"
+          component={PvcsList}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/namespaces"
+          component={NamespacesListPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/pods"
+          component={PodsList}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/storage-classes"
+          component={StorageClassList}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/jobs"
+          component={JobsListPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          path="/clusters/:clusterID/deployments"
+          component={DeploymentsPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          exact
+          path="/clusters"
+          component={ClusterPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          exact
+          path="/databases"
+          component={AdminDBList}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          exact
+          path="/databases/:databaseID"
+          component={AdminDatabaseDetails}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          exact
+          path="/clusters/:clusterID/logs"
+          component={AdminLogsPage}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken && isAdmin}
+          exact
+          path="/clusters/:clusterID/clusterSettings"
+          component={ClusterSettingsPage}
+        />
+        <Route
+          isAllowed={hasToken && isAdmin}
+          exact
+          path="/projects-overview"
+          component={AdminProjectsOverview}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken}
+          exact
+          path="/projects/:projectID/activity"
+          component={ProjectLogs}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken}
+          exact
+          path="/activity"
+          component={UserActivity}
+        />
+        <ProtectedRoute
+          isAllowed={hasToken}
```

Comment:
@khalifan-kfan forgot the admin attribute for this, should also be ```isAllowed={hasToken && isAdmin}```

Reply:
Yes, @neelxie this has been fixed. Thanks